### DISCUSSION TITLE
[BUG?] Model switch disabling/enabling incorrect model

### DISCUSSION THREAD

**Discussion by ghost** [open-webui/open-webui#7461] [Issues]
Hi, I am seeing an issue when it comes to disabling/enabling models in Admin Panel (Settings => models).
I've linked a [video](https://www.youtube.com/watch?v=r7miDHWguLs) showing the issue.

version: v0.4.6
[(latest)](https://github.com/open-webui/open-webui/releases/tag/v0.4.6)
running on docker(Ubuntu 24.04.1) in wsl2.

I've tried it on Debian and performed a clean install, but same thing.

**Issue**
When more than one model is loaded (in the video only 2 are loaded, but I've noticed same thing with 3, 4, and will presume same thing happens further) and clicking the switch on the first model on the list(from top/ bottom) it disables/enables incorrect model.
As you can see in the video it is not a visual bug, can see the requests being made in the network tab as well.

Now, I am running on a 4k monitor, which could be partially to blame? I do not have other monitor so cannot validate, but if anyone could take a look that would be great.
_Edit_
I've dropped the resolution (scaling as well) and issue still persists.

I've tested it in chrome/edge, both behave the same way.

I had one of my mates check it and it works fine for him on a macbook air(safari).

Could any of you take a look and check the behaviour you are getting?

**Comment by tjbck** [comment]
Everything is working as intended here, disabled models are just moving to the bottom. Feel free to verify this yourself with the code.
